src/centralserver/admin-ui/src/views/AppLogin.vue

```diff
@@ -187,63 +186,45 @@ export default (
       this.$refs.form.reset();
       this.loading = true;
 
-      this.login(loginData)
-        .then(
-          () => {
-            // Auth ok. Start phase 2 (fetch user data and current security server info).
-            this.requestUserData()
-              .then(this.fetchServerVersion)
-              .then(this.fetchSystemStatus)
-              .then(() => {
-                this.$router
-                  .replace({
-                    name: RouteName.Members,
-                  })
-                  .catch(swallowRedirectedNavigationError);
-                this.loading = false;
-              })
-              .catch((error) => {
-                this.showError(error);
-                this.loading = false;
-              });
-          },
-          (error) => {
-            // Display invalid username/password error in inputs
-            if (error?.response?.status === 401) {
-              // Clear inputs
-              this.username = '';
-              this.password = '';
-              this.$refs.form.reset();
-
-              // The whole view needs to be rendered so the "required" rule doesn't block
-              // "wrong unsername or password" error in inputs
-              this.$nextTick(() => {
-                // Set inputs to error state
-                this.$refs.form.setErrors({
-                  username: [''],
-                  password: [this.$t('login.errorMsg401') as string],
-                });
-              });
-
-              this.showErrorMessage(this.$t('login.generalError'));
-            } else {
-              this.showError(error);
-            }
-
-            // Clear loading state
-            this.loading = false;
-          },
-        )
-        .catch((error) => {
+      try {
+        await this.login(loginData);
+        await this.fetchServerVersion();
+        await this.fetchSystemStatus();
+        await this.routeToMembersPage();
+      } catch (error) {
+        // Display invalid username/password error in inputs
+        if (error?.response?.status === 401) {
+          // Clear inputs
+          this.username = '';
+          this.password = '';
+          this.$refs.form.reset();
+          // The whole view needs to be rendered so the "required" rule doesn't block
+          // "wrong unsername or password" error in inputs
+          this.$nextTick(() => {
+            // Set inputs to error state
+            this.$refs.form.setErrors({
+              username: [''],
+              password: [this.$t('login.errorMsg401') as string],
+            });
+          });
+          this.showErrorMessage(this.$t('login.generalError'));
+        } else {
           this.showError(error);
-          this.loading = false;
-        });
+        }
+        // Clear loading state
+        this.loading = false;
+      }
     },
     async requestUserData() {
```

Comment:
requestUserData is not called at all, but it should be, right after this.login().
Now after the login there is no permissions data etc. and the tabs are not visible.

Reply:
Fixed in https://github.com/nordic-institute/X-Road/pull/1163/commits/8a6d39152883774f111a1b8b2b149fb5c9e0d4ba